snmp/CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 
 ## Unreleased
 
+***Fixed***:
+
+* Fix metrics mappings ([#15668](https://github.com/DataDog/integrations-core/pull/15668))
```

Comment:
Out of curiosity, what is being fixed? It might be obvious to the maintainers of SNMP but I can't quite understand so it might be nice to make this entry slightly more detailed

Reply:
We are fixing the values of the mappings.
For some profiles, some mappings have wrong values and for others we didn't have the full list of values.

Reply:
addressed in [cbf51e4](https://github.com/DataDog/integrations-core/pull/15668/commits/cbf51e46ce3f2e97fd0f4e33c2bee8ca134e30e7)